app/styles/components/speaker-list.scss

```diff
@@ -13,3 +13,19 @@
 .word-break {
   overflow-wrap: break-word;
 }
+
+.thumbnail-square {
```

Comment:
Also this is very outdated the article belongs to 2014. Today we have object fit property which may do the work. Implement using it.

Reply:
Leave it , @maze-runnar Just fix the warnings this will be handled in a follow up PR.